kalite/updates/views.py

```diff
@@ -27,7 +27,9 @@ def update_context(request):
 @render_to("updates/update_videos.html")
 def update_videos(request, max_to_show=4):
     context = update_context(request)
-    messages.warning(request, _('For low-powered devices like the Raspberry Pi, please download videos one at a time.'))
+    messages.warning(request, _('For low-powered devices like the Raspberry Pi, please download less than 25 videos at a time.'))
```

Comment:
Wondering a bit why this is included, but I previously had this issue opened because the message is very odd anyways, because it's difficult to understand exactly what it means: #3782